src/components/common/Bar.vue

```diff
@@ -2,8 +2,16 @@
   <div v-if="showMessage" :class="`bar ${barType}`">
     <p>
       <slot />
+      <span
+        >Come visit
+        <span v-if="link" class="link" @click="goToLink(link)">us!</span>
```

Comment:
you want to always show "Come visit us!"?

Reply:
This is only in staging Bar ;)

Reply:
ah no, you are right. Yesterday I must have been so sleepy 😅